app/views/campsites/edit.html.erb

```diff
@@ -1,10 +1,14 @@
+<h2>Edit <%= @campsite.name %> </h2>
+
 <%= form_with url: "/campsites/#{@campsite.id}", method: :patch, local: true do |form| %>
-  <%= form.label :name %>
-  <%= form.text_field :name %>
-  <%= form.label :tent_only %>
-  <%= form.text_field :tent_only %>
-  <%= form.label :overnight_fee %>
-  <%= form.text_field :overnight_fee %>
+  <%= form.label :name, 'Name:' %>
+  <%= form.text_field :name, value: "#{@campsite.name}" %><br><br>
+  <%= form.label :overnight_fee, 'Overnight Fee:' %>
+  <%= form.text_field :overnight_fee, value: "#{@campsite.overnight_fee}" %><br><br>
+  <%= form.label :tent_only, 'Tent Only:' %>
+  <%= form.check_box :tent_only, {checked: true}, 'true', 'false' %><br><br>
```

Comment:
This will make it so the checkbox to always be checked, regardless of whether the original record was true or false.  Is that the intention?